vmm/src/igvm/igvm_loader.rs

```diff
@@ -136,6 +174,48 @@ pub fn load_igvm(
     cmdline: &str,
     #[cfg(feature = "sev_snp")] host_data: &Option<String>,
 ) -> Result<Box<IgvmLoadedInfo>, Error> {
+    let hypervisor_type = cpu_manager.lock().unwrap().hypervisor_type();
+    let isolated_page_size_4kb: u32 = match hypervisor_type {
+        #[cfg(feature = "mshv")]
+        HypervisorType::Mshv => mshv_bindings::hv_isolated_page_size_HV_ISOLATED_PAGE_SIZE_4KB,
+        #[cfg(feature = "kvm")]
+        HypervisorType::Kvm => 0x1000,
+    };
+    let normal_page_type: u32 = match hypervisor_type {
+        #[cfg(feature = "mshv")]
+        HypervisorType::Mshv => mshv_bindings::hv_isolated_page_type_HV_ISOLATED_PAGE_TYPE_NORMAL,
+        #[cfg(feature = "kvm")]
+        HypervisorType::Kvm => 1,
```

Comment:
Can these magic numbers be defined somewhere? Perhaps they should be added to `kvm_bindings`?

This can happen separately though.